changes/1414-silversurfer34.md

```diff
@@ -0,0 +1,2 @@
+Fix proposal for the issue 1414, wrong behavior of json method when there are nested models with custom root.

```

Comment:
the change summary should be one line (or paragraph). Please remove "Fix proposal for the issue 1414", this for the change summary, not a description of the PR.

You can also make this much more concise.

Reply:
done